src/us/kbase/common/utils/CondorUtils.java

```diff
@@ -239,15 +240,20 @@ public static String submitToCondorCLI(String ujsJobId, AuthToken token, String
         String jobID = null;
         int retries = 10;
 
+        String stderr = null;
         while (jobID == null && retries > 0) {
-            jobID = runProcess(cmdScript).stdout.get(0);
+            CondorResponse r = runProcess(cmdScript);
+            if (r.success)
+                jobID = r.stdout.get(0);
+
+            stderr = String.join(", ", r.stderr);
             retries--;
```

Comment:
would it make sense to add some time delay between attempts here?

Reply:
There is already a time delay in running condor_q already of 1-3 seconds. I would have to decrease the number of retries if I put a delay in.